root/relationship/linktype/RelationshipTypeIndex.js

```diff
@@ -0,0 +1,232 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {sortBy, upperFirst} from 'lodash';
+
+import {withCatalystContext} from '../../context';
+import Layout from '../../layout';
+import Cardinality from '../../static/scripts/common/components/Cardinality';
+import OrderableDirection
+  from '../../static/scripts/common/components/OrderableDirection';
+import Relationship
+  from '../../static/scripts/common/components/Relationship';
+import linkedEntities from '../../static/scripts/common/linkedEntities';
+import expand2react from '../../static/scripts/common/i18n/expand2react';
+import formatEntityTypeName
+  from '../../static/scripts/common/utility/formatEntityTypeName';
+
+type Props = {
+  +$c: CatalystContextT,
+  +relType: LinkTypeT,
+};
+
+const RelationshipTypeIndex = ({
+  $c,
+  relType,
```

Comment:
Up to you if you think this change makes sense, but we do tend to use `linkType` more often for naming variables that refer to link types, since that's how they're called everywhere else. Though the page/component name is  `RelationshipTypeIndex` so I can see how that follows from that, too
```suggestion
  linkType,
```

Reply:
I personally prefer relType, but that's because I'd like to eventually get rid to references in the code to stuff we no longer use anywhere else. I could change to linkType for now and change everything together eventually though, if you prefer...